Gems/ROS2/Code/Source/VehicleDynamics/Utilities.cpp

```diff
@@ -191,4 +191,25 @@ namespace ROS2::VehicleDynamics::Utilities
         return AZ::EntityComponentIdPair(wheelEntityId, hingeComponent->GetId());
     }
 
+    float RampVelocity(float targetVelocty, float lastVelocity, AZ::u64 deltaTimeNs, float acceleration, float maxVelocity)
+    {
+        const float eps = 0.01f * acceleration; // minimal difference in speed to apply acceleration
```

Comment:
Why not make `eps = deltaTimeSec * acceleration;`? That way it will not apply the step only if it's less than the amount of the step itself. Using `0.01f * acceleration` could be far off from the step and see that lastVelocity only advance in chunks after reaching `0.01f * acceleration`.